.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gir"]
 	path = gir
-	url = https://github.com/gtk-rs/gir
+	url = https://github.com/sdroege/gir
```

Comment:
We'll need to not commit that too. :) (you know that you have an option in `generator.py` to run another `gir` right? ;) )

Reply:
Yes but then the CI fails, and if a single CI job fails it's cancelling all the others so I wouldn't know if all are actually passing :)